selftests/unit/test_utils_partition.py

```diff
@@ -45,12 +37,8 @@ class TestPartition(unittest.TestCase):
 
     @unittest.skipIf(missing_binary('mkfs.ext2'),
                      "mkfs.ext2 is required for these tests to run.")
-    @unittest.skipIf(missing_binary('sudo'),
+    @unittest.skipIf(not process.can_sudo(),
```

Comment:
We can check if the execution of the needed command will succeed, which is a lot closer to the execution of `id -u`, which I consider a valid fallback. If we support an specific command, tests like this would not be skipped under configurations like the following (example based on Fedora's default configuration):

```
Cmnd_Alias STORAGE = /sbin/fdisk, /sbin/sfdisk, /sbin/parted, /sbin/partprobe, /bin/mount, /bin/umount, /sbin/mkfs.ext2
cleber  ALL=(ALL) NOPASSWD: STORAGE
```

This setup would allow most tests that require privileged disk operations to succeed, with few or none false positivies.

The only configuration possible where a user would have success without the option to specify the command would be:

```
cleber ALL=(ALL) NOPASSWD: ALL
```